helper.php

```diff
@@ -151,7 +152,7 @@ public function replaceTags($id, $user, $tags)
         }
 
         $db = $this->getDB();
-        $db->query('BEGIN TRANSACTION');
+        $db->exec('BEGIN TRANSACTION');
```

Comment:
Just a quick note: you could also use `$db->getPDO()->beginTransaction()` but I don't think it makes a difference.